style.css

```diff
@@ -0,0 +1,34 @@
+body{
```

Comment:
add css resetting rule at the top 
*{
margin: 0;
padding: 0;
box-sizing: border-box;
}